README.md

```diff
@@ -1 +1,141 @@
+selected one. When clicking on one of the countries cards the user is redirected to the 'Details' page.
+- Details page: The 4 more relevants cities are displayed. Clicking the 'Left arrow' on the navbar redirects the user
+to the homepage.
+
+=======
+
+## Built With 
+
+- HTML/SCSS
+- React.JS
+- Redux
+- Jest
+- Node.JS
+
+=======
+
+### Project requirements
+
+Need a modern browser to visit the live site.
+
+
+## Getting Started
+
+If you want a copy of this file go to the github repository and download it from there
+
+- [`git clone https://github.com/Metsanpeitto/Capstone-3_Redux`](https://github.com/Metsanpeitto/Capstone-3_Redux)
+
+### Presentation video
+
+[Video](https://drive.google.com/file/d/17YIk7TeuBCjYPRM0oDl_YhMJ57uQmpRW/view?usp=sharing)
+
+
+### Live
+
+[Live](https://metsanpeitto.github.io/Capstone-3_Redux/#/)
+
+
+### Setup
+
+```cmd
+git clone  https://github.com/Metsanpeitto/Capstone-3_Redux.git
+cd ./Capstone-3_Redux
+```
+
+
+### Install
+
+```cmd
+npm install
+```
+
+### Usage
+
+```cmd
+npm start
+```
+
+### Verify linters
+
+```cmd
+npx eslint .
+npx stylelint "**/*.{css,scss}"
+```
+- Autofix linters errors
+
+```cmd
+npx eslint-fix
+npx stylelint-fix
+```
+
+## To run jest test
+```cmd
+npm run test
+npm run jest
+
+
+If you are non a technical person you can download the zip file.
+
+- To do that you have to go to the green button that says 'Code' and then press on it.
+- Choose the option 'Download Zip' and wait until it download.
+- Then you need to decompress it.
+- You will need to have Zip compress file software installed in your computer. If you don't have it you can download it from here
+  [`https://www.7-zip.org/download.html`](https://www.7-zip.org/download.html)
+- Do double click on the Capstone-1.zip and choose a folder where to place all teh decompressed files.
+- Once you are done decompressing, you can open the created folder.
+- In it you can double click on the index.html file and choose a browser to open it (For example google Chrome Browser).
+- Now you should be able to see the project.
+- Remember to use the mouse to navigate through the website. Also you can use the keys 'arrow up' and 'arrow down' of your keyboard
+  to scroll up and down.
+
+
+### Prerequisites
+
+- Node.JS
+
+=======
+
+## Author
+
+👤 **Abraham Rodriguez**
+
+- GitHub: [@metsanpeitto](https://github.com/Metsanpeitto)
+- LinkedIn: [Abraham Rodriguez](https://www.linkedin.com/in/abraham-rodriguez-3283a319a/)
+- Portfolio: [Waldenberg](https://portfolio.waldenberginc.com)
+
+=======
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+=======
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+=======
+
+## Acknowledgments
+
+- Microverse for giving me this chance
+- The amazing code reviewers for making me improve every day :thumbsup:
+
```

Comment:
- [ ] The Acknowledgements section should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Perfect! I have already fixed it. Thanks for your time :1st_place_medal: